<launch>
  <node name="talker1" pkg="map_mux" type="talker.py" /> 

  <!--<arg name="map_file1" default="/nishome/aunwala/catkin_ws/src/bwi_common/utexas_gdc/maps/3ne.yaml" />-->
  <!--<arg name="map_file2" default="/nishome/aunwala/Downloads/3ne-real-new.yaml" />-->
  <!--<arg name="map_file3" default="/nishome/aunwala/Downloads/3ne-real-new.yaml" />-->

  <arg name="map_file1" default="/home/goatfigpc/catkin_ws/src/map_mux/maps/3ne-real-new.yaml" />
  <arg name="map_file2" default="/home/goatfigpc/catkin_ws/src/map_mux/maps/map11.yaml" />
  <arg name="map_file3" default="/home/goatfigpc/catkin_ws/src/map_mux/maps/map11.yaml" />
  
  <node name="map_server1" pkg="map_server" type="map_server" args="$(arg map_file1)"> 
    <remap from="map" to="map1"/>
  </node>

  <node name="map_server2" pkg="map_server" type="map_server" args="$(arg map_file2)"> 
    <remap from="map" to="map2"/>
  </node>

  <node name="map_server3" pkg="map_server" type="map_server" args="$(arg map_file3)"> 
    <remap from="map" to="map3"/>
  </node>

  <node name="map_mux1" pkg="map_mux" type="main.py" /> 

</launch>
